crates/workspace/src/toolbar.rs

```diff
@@ -105,12 +105,15 @@ impl Render for Toolbar {
             .bg(cx.theme().colors().toolbar_background)
             .child(
                 h_flex()
+                    .min_h(rems_from_px(24.))
+                    .items_center()
                     .justify_between()
                     .gap(Spacing::Large.rems(cx))
                     .when(has_left_items, |this| {
                         this.child(
                             h_flex()
                                 .flex_auto()
+                                .items_center()
```

Comment:
We shouldn't need `items_center` here, since that is applied by `h_flex`:

https://github.com/zed-industries/zed/blob/8c54a46202bbffdbc452828f80fffd1b69819b8c/crates/ui/src/styled_ext.rs#L26